tests/Doctrine/Tests/DBAL/Functional/DataAccessTest.php

```diff
@@ -433,12 +434,33 @@ public function testNativeArrayListSupport()
         $this->assertEquals(5, count($data));
         $this->assertEquals(array(array(100), array(101), array(102), array(103), array(104)), $data);
 
+        $stmt = $this->_conn->executeQuery('SELECT test_int FROM fetch_table WHERE test_int IN (?)',
```

Comment:
Ok I could, but this is also a good way to prove the code is fully backward compatible